apps/omg_watcher/lib/omg_watcher/exit_processor/update_db_cores/new_exits.ex

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.ExitProcessor.UpdateDB.NewExits do
+  @moduledoc """
+  Functions related to the logic on updating DB for ExitProcessor.
+  """
+
+  alias OMG.Watcher.ExitProcessor.ExitInfo
+
+  @doc """
+  DB updates needed to add new exits from Ethereum events into tracked state.
+
+  The list of `exit_contract_statuses` is used to track current (as in wall-clock "now", not syncing "now") status.
+  This is to prevent spurious invalid exit events being fired during syncing for exits that were challenged/finalized
+  Still we do want to track these exits when syncing, to have them spend from `OMG.State` on their finalization
+  """
+  @spec get_db_update(list(map()), list(map)) :: {:ok, list()} | {:error, :unexpected_events}
+  def get_db_update(new_exits, exit_contract_statuses)
```

Comment:
Copy of the `new_exits` function without using `state` from `Core`: 
https://github.com/omisego/elixir-omg/blob/d118f210b41c49aa29d5f5e2e1bdea70c5ab9fa2/apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex#L176-L198